frame/transaction-payment/asset-tx-payment/src/tests.rs

```diff
@@ -479,20 +608,24 @@ fn post_dispatch_fee_is_zero_if_pre_dispatch_fee_is_zero() {
 			let caller = 333;
 			let beneficiary = <Runtime as system::Config>::Lookup::unlookup(caller);
 			let balance = 100;
+
 			assert_ok!(Assets::mint_into(asset_id.into(), &beneficiary, balance));
 			assert_eq!(Assets::balance(asset_id, caller), balance);
+
 			let weight = 1;
 			let len = 1;
-			// we convert the from weight to fee based on the ratio between asset min balance and
-			// existential deposit
 			let fee = (base_weight + weight + len as u64) * min_balance / ExistentialDeposit::get();
```

Comment:
This looks old school, should be updated to use `AssetConversion::quote_price_tokens_for_exact_tokens`?